zerver/openapi/zulip.yaml

```diff
@@ -8202,6 +8202,10 @@ paths:
         This endpoint mutes/unmutes a topic within a stream that the current
         user is subscribed to. Muted topics are displayed faded in the Zulip
         UI, and are not included in the user's unread count totals.
+
+        **Changes**: New in Zulip 7.0 (feature level 168), No error is raised
+        when an `add` operation is requested for a topic that has already been muted.
+        Previously, the error `Topic is already muted` was raised.
```

Comment:
"error is raised" is normal language for Python code, but in the API context, we're not "raising an error/exception", we're returning an HTTP 200 success response or an HTTP 400 error. 

Also the "New in Zulip 7.0" language should only be used if the feature is new in that version -- which it isn't, we're just documenting a small change.

So this should say something like "**Changes**. Before Zulip 7.0 (feature level 168), this endpoint returned an error if asked to mute a topic that was already muted.".

The string isn't worth mentioning, because the string is tagged for translation, so a correct client shouldn't be inspecting the string at all; we use `code` for errors that we want to be parsed by clients.